reflekt-plugin/src/main/kotlin/org/jetbrains/reflekt/plugin/analysis/models/Instances.kt

```diff
@@ -0,0 +1,27 @@
+package org.jetbrains.reflekt.plugin.analysis.models
+
+import org.jetbrains.kotlin.psi.*
+import org.jetbrains.reflekt.plugin.analysis.processor.FileID
+import org.jetbrains.reflekt.plugin.analysis.processor.source.instances.*
+
+typealias TypeInstances<T> = MutableList<T>
+typealias ObjectInstances = TypeInstances<KtObjectDeclaration>
+typealias ClassInstances = TypeInstances<KtClass>
+typealias FunctionInstances = TypeInstances<KtNamedFunction>
```

Comment:
do we really need typealiases for a mutable list? sometimes it's so tedious to always navigate to the definition just to see it's a list of something...

Reply:
Yes, it is used also in ClassOrObjectInstancesProcessor for example. Currently it is a list, but I am not sure tat we keep it in the future